sources/PreferenceInfo.h

```diff
@@ -12,8 +12,10 @@ typedef NS_ENUM(NSInteger, PreferenceInfoType) {
     kPreferenceInfoTypeCheckbox,
     kPreferenceInfoTypeInvertedCheckbox,  // true=checked, false=unchecked. Handy when inverting a checkbox's text, but the user defaults key can't be changed.
     kPreferenceInfoTypeIntegerTextField,
+    kPreferenceInfoTypeUnsignedIntegerTextField,
     kPreferenceInfoTypeStringTextField,
     kPreferenceInfoTypePopup,
+    kPreferenceInfoTypeUPopup,
```

Comment:
Let's avoid abbreviations and call it UnsignedIntegerPopup. Also, add a comment to Popup saying it is deprecated in favor of UnsignedIntegerPopup.